pkg/api/v1/conversion.go

```diff
@@ -44,6 +44,7 @@ func addConversionFuncs() {
 			case "metadata.name",
 				"metadata.namespace",
 				"status.phase",
+				"status.podIP",
```

Comment:
If you want to filter based on this field in the client, there are client changes that need to be made, also.